dist/perPackageTabs.js

```diff
@@ -0,0 +1,30 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I could name this `package.js`to match the `package.css` and `package.html` but there is also `packages.html`, `packages.css` and `packages.js` The package(s) ones are for when you click on "Browse Packages" and it shows the list of packages. 

I don't know what the convention is but it would be nice if the javascript files were named according to their function instead of who loads them.